rds-deployment-existing-HA/azuredeploy-HA.json

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
@AshishSharma303 Please see the contribution guide on the files needed for the PR to be merged https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE